src/ConfigurationParser.cpp

```diff
@@ -1,6 +1,7 @@
 #include "ConfigurationParser.hpp"
 
 #include <stout/foreach.hpp>
+#include <map>
```

Comment:
Why ?

Reply:
just for linting, I could have skipped that